app/code/core/Mage/Customer/Model/Convert/Parser/Customer.php

```diff
@@ -696,10 +606,9 @@ public function parse()
                         if ($customer->getDefaultBilling()) {
                             $model->setDefaultBilling($customer->getDefaultBilling());
                         } else {
-                            $shippingAddress->save();
+                            $billingAddress->save();
```

Comment:
This is a fundemental code change. 

Even if this is right, it should not be in this commit.

Reply:
Will be a own PR.

Reply:
Resolved in #701